drivers/d3d12/rendering_device_driver_d3d12.h

```diff
@@ -258,9 +269,9 @@ class RenderingDeviceDriverD3D12 : public RenderingDeviceDriver {
 	uint32_t res_barriers_count = 0;
 	uint32_t res_barriers_batch = 0;
 #ifdef DEV_ENABLED
```

Comment:
The correct ifdef condition for this seem to be `DEBUG_COUNT_BARRIERS` not `DEV_ENABLED` (probably won't need `[[maybe_unused]]` if it's used).

Reply:
That's technically true, but the define needed to be moved to the header file first (despite being commented out).